corpus/config_db.json

```diff
@@ -1,10 +1,63 @@
 [
+  {
+    "fields": {
+      "about": "The NITK-IEEE Computer Society is a conglomeration of students with a passion for Computer Science and Information Technology. This SIG is a great platform for those who want to gain expertise in computer based technology.\r\nIn the true spirit of divide and conquer, our community intends to explore and contribute to the rapidly evolving world of computer technologies by carrying out research and projects in the broad domains of computer science, in the form of 3 research groups : Intelligence, Systems and Core Computer Science.",
+      "name": "CompSoc",
+      "slug": "compsoc",
+      "what_we_do": "Intelligence deals with fields such as Machine Learning, Artificial Intelligence, Deep Learning, Computer Vision, NLP. Systems is related to fields involving Operating Systems, Parallel Computing, Network Security, Distributed Systems. Core Computer Science involves Algorithms, Cryptography, Databases, Game Theory, Web & Software Development ."
+    },
+    "model": "config.sig",
+    "pk": 1
+  },
+  {
+    "fields": {
+      "about": "The NITK Diode SIG is a combination of the Circuits and Systems (CAS) Society, Signal Processing Society (SPS) and the Industrial Applications Society (IAS).\r\nThe SIG consists of a team of students with a strong interest in the field of Electronics and Electrical Science, and works on conceiving and pioneering solutions to fundamental and applied problems in the fields of robotics, microcontrollers and signal processing.\r\nIEEE NITK inaugurated the IEEE Industrial Applications Society student chapter in the year 2019. This year we saw a steady increase in the number of activities and events organized by the student chapter.",
+      "name": "Diode",
+      "slug": "diode",
+      "what_we_do": "For 2019-20 IAS saw one project on a battery based solar powered inverter being completed under its wing. To enrich its members with knowledge in the fields of power electronics, the chapter held 3 workshop sessions. Additionally, in collaboration with the Department of Electrical and Electronics Engineering, the IAS student chapter organized a national conference PETPES 2019."
```

Comment:
For the 2022-23 academic year, IAS showcased a project on a three-level boost converter design for MPPT applications in PV cells. SPS also boasts to showcase state-of-the-art projects such as 'Low Light Image Enhancement using MIRNet', and 'Leveraging Frequency Analysis for Deep Fake Image Classification' at the annual project expo at NITK. In addition, SPS finds its presence in paper discussion sessions, expert talks, and many more enriching activities. Nation wide Workshops, Knowledge sharing sessions and Hackathons were conducted throughout the year by the all three chapters under Diode.

Reply:
done